concrete/src/Cache/Page/PageCache.php

```diff
@@ -24,21 +21,22 @@
     /**
      * Build a Response object starting from a cached page.
      *
-     * @param \Concrete\Core\Cache\Page\PageCacheRecord $record The cache record containing the cached page data.
+     * @param \Concrete\Core\Cache\Page\PageCacheRecord $record the cache record containing the cached page data
```

Comment:
Why start this sentence with a lower case character? The capitalized character helps my eyes jump to the start of the statement